server/vulnerabilities/nvd/sync/cve_syncer.go

```diff
@@ -392,6 +469,218 @@ func (s *CVE) sync(ctx context.Context, lastModStartDate *string) (newLastModSta
 	return newLastModStartDate, nil
 }
 
+func (s *CVE) DoVulnCheck(ctx context.Context) error {
+	vulnCheckArchive := "vulncheck.zip"
+	baseURL := "https://api.vulncheck.com/v3/backup/nist-nvd2"
```

Comment:
Is this updated as frequently as their API feed?

Reply:
No official documentation, but in my testing it seems to be up to date.